src/schema/alerts.ts

```diff
@@ -207,29 +207,35 @@ export const updateAlerts = async (
 
 export const getAlerts = async (
   con: DataSource,
-  userId: string,
+  userId?: string,
 ): Promise<Omit<Alerts, 'flags'>> => {
-  const alerts = await con.getRepository(Alerts).findOneBy({
-    userId,
-  });
+  const alerts = userId
+    ? await con.getRepository(Alerts).findOneBy({
+        userId,
+      })
+    : undefined;
```

Comment:
Instead of a ternary here, could we do an early return if there is no userId?

Reply:
Usually yes, but since we return `ALERTS_DEFAULT` on the bottom I just left it like this so the rest of the code is the same. I would have to return `ALERTS_DEFAULT` twice.

Reply:
I see. Shouldn't the findOneBy return null/undefined if it doesn't find anything?

Reply:
yeah, but its a type issue where it can do one or the order, and also we don't want to even run a query if no userId